mopidy/local/__init__.py

```diff
@@ -106,26 +106,12 @@ def get_images(self, uris):
         """
         Lookup the images for the given URIs.
 
-        The default implementation will simply call :meth:`lookup` and
-        try and use the album art for any tracks returned. Most local
-        libraries should replace this with something smarter or simply
-        return an empty dictionary.
+        The default implementation will simply return an empty dictionary.
 
         :param list uris: list of URIs to find images for
         :rtype: {uri: tuple of :class:`mopidy.models.Image`}
         """
-        result = {}
-        for uri in uris:
-            image_uris = set()
-            tracks = self.lookup(uri)
-            # local libraries may return single track
-            if isinstance(tracks, models.Track):
-                tracks = [tracks]
-            for track in tracks:
-                if track.album and track.album.images:
-                    image_uris.update(track.album.images)
-            result[uri] = [models.Image(uri=u) for u in image_uris]
-        return result
+        return {}
```

Comment:
Should we add a proper implementation to the JSON library? Or should we ignore this and focus on merging local and local-sqlite?

Reply:
I think -local-json has outlived it's usefulness and should probably be removed. So I'm OK with not solving this there. Also mopidy-local-images is probably better for this anyway.